forecast.js

```diff
@@ -0,0 +1,41 @@
+var request = require('request');
+
+module.exports = function (location) {
+  return new Promise (function (resolve, reject) {
+    var apiKey = process.env.API_KEY;
+    console.log(location);
+    var url = `http://api.openweathermap.org/data/2.5/forecast?q=${encodeURIComponent(location)}&mode=json&units=imperial&APPID=${apiKey}`;
+
+    if (!location) {
+      reject('No Location Provided');
+    };
+
+    request({
+      url,
+      json: true
+    }, function (error, response, body) {
+      if (error) {
+        reject('Unable to fetch Weather: ', error.message);
+      } else { // Print entire JSON 'nicely': console.log(JSON.stringify(body, null, 4));
+        var location = body.city.name;
```

Comment:
Your code was only showing the next days temperature at midnight. I updated it to show the next 5 days at 3PM. This code needs to be refactored badly. Feel free to start a new PR for the refactoring!